### PR TITLE
[release/3.1] Fix the badness and get this branch building

### PR DESCRIPTION
**Pull request by dougbu**
- work around dotnet/core-eng#7970 w/ lots of logging (ugly hack that should be reverted ASAP)
- work around dotnet/arcade#4021 (not too bad)

### CODE REVIEW

**Comment by dougbu** [comment]
Success modulo a missing `New-Item -ItemType Directory .\.tools\ -ErrorAction SilentlyContinue`❕❕ 

/bump reviewers (should get this in ASAP i.e. without waiting for a merge from the 3.0 branch).

File: eng/scripts/InstallTar.ps1 [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+elseif ((Test-Path $tarCommand) -And (-Not $Force)) {
+    Write-Verbose "Repo-local Git installation and $tarCommand already exist, skipping Git install."
+}
+else {
+    if (-not $GitVersion) {
+        $globalJson = Get-Content "$repoRoot\global.json" | ConvertFrom-Json
+        $GitVersion = $globalJson.tools.Git
+    }
+
+    $Uri = "https://netcorenativeassets.blob.core.windows.net/resource-packages/external/windows/git/Git-${GitVersion}-64-bit.zip"
```
Comment (Pilchie) [inline eng/scripts/InstallTar.ps1]: If this 64-bit version of `tar` from the `git` package works, just running the `$env:SystemRoot\SysNative\tar.exe` work instead?

**Review by Pilchie — APPROVED** [review]

File: eng/scripts/InstallTar.ps1 (hunk 1 above)
Comment (dougbu) [reply eng/scripts/InstallTar.ps1]: That would also work but _only_ in a 32bit application (`SysNative` doesn't exist when running in a 64 bit process). This way all bitnesses are covered.

Comment (Pilchie) [reply eng/scripts/InstallTar.ps1]: I hate the WOW system.

**Pull request merged**